### PR TITLE
Feature/antenna led gpg3

### PR DESCRIPTION
**Pull request by CleoQc**
Add an override in Raspbian For Robots for the antenna led. 
In R4R, the antenna LED on the GPG3 is yellow once novnc is started, and a wifi connection is active.  This dependency doesn't exist on regular Raspbian.

Also the colour is yellow so that - from a support point of view - we can differentiate between Raspbian and Raspbian for Robots.

Goes hand in hand with https://github.com/DexterInd/GoPiGo3/pull/224

### CODE REVIEW

File: upd_script/antenna_wifi_override.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+[Unit]
```
Comment (RobertLucian) [inline upd_script/antenna_wifi_override.conf]: Take out one `[Unit]`. It's redundant having 2 of them.

File: upd_script/antenna_wifi_override.conf [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Description=R4R Antenna Wifi Indicator 
+Requires=novnc.service
+After=novnc.service
+
+[Service]
+ExecStart=
```
Comment (RobertLucian) [inline upd_script/antenna_wifi_override.conf]: Likewise for `ExecStart`. Only one is required.

File: upd_script/fetch.sh [hunk 3]
```diff
@@ -90,6 +90,11 @@ update_gopigo() {
         feedback "##############################"
         curl -kL https://raw.githubusercontent.com/DexterInd/GoPiGo3/$selectedbranch/Install/update_gopigo3.sh | sudo -u pi bash -s -- --bypass-rfrtools  $selectedbranch
 
+        # set wifi antenna led for Raspbian for Robots
+        sudo mkdir /etc/systemd/system/antenna_wifi.service.d/
```
Comment (RobertLucian) [inline upd_script/fetch.sh]: It has to be `sudo mkdir -p ...`.

File: upd_script/antenna_wifi_override.conf (hunk 1 above)
Comment (CleoQc) [reply upd_script/antenna_wifi_override.conf]: woah, how did that extra one get in??

File: upd_script/antenna_wifi_override.conf (hunk 2 above)
Comment (CleoQc) [reply upd_script/antenna_wifi_override.conf]: Not based on my testing and what I read. If you just have one, then the ExecStart is additive and then it fails. 
The first one is meant to cancel the one in the other service.

File: upd_script/fetch.sh (hunk 3 above)
Comment (CleoQc) [reply upd_script/fetch.sh]: I guess for safety but if /etc/systemd/system doesn't already exist, we have bigger problems

File: upd_script/antenna_wifi_override.conf (hunk 1 above)
Comment (CleoQc) [reply upd_script/antenna_wifi_override.conf]: sneaky VS code was hiding line 1 from me.  Fixed now.

File: upd_script/antenna_wifi_override.conf (hunk 2 above)
Comment (CleoQc) [reply upd_script/antenna_wifi_override.conf]: See https://askubuntu.com/questions/659267/how-do-i-override-or-configure-systemd-services
_I had to explicitly clear ExecStart before setting it again, as it is an additive setting, similar to After, Environment (as a whole, not per-variable) and EnvironmentFile, and opposed to overriding settings like RestartSec or Type. ExecStart can have multiple entries only for Type=oneshot services._

My testing was giving me exactly that behaviour

**Pull request merged**